src/stdlib/CryptoUtils.scillib

```diff
@@ -0,0 +1,33 @@
+(* ******************************************************************************** *)
+(*   This file is part of scilla.                                                   *)
+(*                                                                                  *)
+(*   Copyright (c) 2018 - present Zilliqa Research Pvt. Ltd.                        *)
+(*                                                                                  *)
+(*   scilla is free software: you can redistribute it and/or modify it under the    *)
+(*   terms of the GNU General Public License as published by the Free Software      *)
+(*   Foundation, either version 3 of the License, or (at your option) any later     *)
+(*   version.                                                                       *)
+(*                                                                                  *)
+(*   scilla is distributed in the hope that it will be useful, but WITHOUT ANY      *)
+(*   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR  *)
+(*   A PARTICULAR PURPOSE.  See the GNU General Public License for more details.    *)
+(*                                                                                  *)
+(*   You should have received a copy of the GNU General Public License along with   *)
+(*   scilla.  If not, see <http://www.gnu.org/licenses/>.                           *)
+(* ******************************************************************************** *)
+scilla_version 0
+
+library CryptoUtils
+
+(* group order for the bn128_G1 curve *)
+let alt_bn128_G1_q = Uint256 21888242871839275222246405745257275088548364400416034343698204186575808495617
+
+(* generator for the bn128_G1 curve *)
+let alt_bn128_G1_g =
+  let one = 0x0000000000000000000000000000000000000000000000000000000000000001 in
+  let two = 0x0000000000000000000000000000000000000000000000000000000000000002 in
+  Pair {ByStr32 ByStr32} one two
+
+let alt_bn128_G1_zero =
+  let zero = 0x0000000000000000000000000000000000000000000000000000000000000000 in
+  Pair {ByStr32 ByStr32} zero zero
```

Comment:
Haliq, could you please add a link explaining these numbers for future reference? Everything else LGTM.